exonum_client/protobuf_loader.py

```diff
@@ -142,10 +157,20 @@ def initialize(self) -> None:
         # Add a directory with exonum_modules into the Python path:
         sys.path.append(self._proto_dir)
 
+        getLogger(__name__).debug(
```

Comment:
Maybe it makes sense to implement `__repr__` for `ExonumClient` and here use formatting via "%s", so the debug log entry won't affect the performance if enabled log level is higher than `debug` (it's ok to have non-lazy string formatting if the case is unlikely, but for the common scenarios it's better to do less unnecessary work).

The same applies to other places where there is an eager string formatting for OK scenarios.

Reply:
Implementation of `__repr__` is a good idea. I'm not sure about the ormatting via "%s", though, since f-strings are better in performance (as far as I know).

Reply:
Logging implementation does not perform string formatting if not needed (e.g. `debug` log entry with `info` log output), so with "%s" string there will be **no formatting at all** if it's not required. With f-strings formatting will happen even if the log entry will not be created.

Reply:
Here I mean not `logging.debug("Some_val %s" % "42")`, but `logging.debug("Some_val %s", "42")`.